newproject01/newproject01/q1.c

```diff
@@ -0,0 +1,67 @@
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
Add a blank line between the include and the defines

```
#define _CRT_SECURE_NO_WARNINGS

#include <stdio.h>

#define N 8
```

Reply:
done

Reply:
Not Done 😄

Reply:
done :)